src/app/personside/infotabs/ytelser/felles-styling/EkspanderbartYtelserPanel.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import { EkspanderbartpanelBase } from 'nav-frontend-ekspanderbartpanel';
+import styled from 'styled-components';
+import { Bold } from '../../../../../components/common-styled-components';
+import { Normaltekst } from 'nav-frontend-typografi';
+
+interface Props {
+    children: ReactNode;
+    tittel: string;
+    tittelTillegsInfo?: (string | null)[];
```

Comment:
Hva er tanken bak nullable entries her?

Reply:
Noen av feltene som ble sendt inn her kom fra backenden og kunne være null, blir mindre jobb å håndtere det samlet her

Reply:
Njoa, men det gir jo egentlig ikke mening for den komponenten at den får inn nullable entries. Selv om du gjenbruker kode. Men null ztress for meg